ansible/my_ansible_playbook.yml

```diff
@@ -0,0 +1,68 @@
+- name: Update Managed Hosts
+  hosts: managedhosts
+  remote_user: ec2-user
+
+  vars_prompt:
+
+  - name: trello_api_key
+    prompt: Enter your Trello API key..
+  
+  - name: trello_api_token
+    prompt: Enter your Trello API Token..
+
+  tasks:
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes
+
+  - name: Install Python3
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+    become: yes
+
+  - name: Install Poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+    args:
+      creates: /home/ec2-user/.local/bin/poetry
+
+  - name: Create todoapp Directory
+    ansible.builtin.shell: mkdir /opt/todoapp
+    args:
+      creates: /opt/todoapp
+    become: yes
+    
+  - name: Change folder owner to ec2-user
+    ansible.builtin.shell: chown ec2-user /opt/todoapp
+    become: yes
```

Comment:
Worth noting that we can do these two directly with the [file module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html) - it's not adding masses of value but it does help us rely on the future extensions and edge cases they might have considered:
```yml
- name: Create todoapp Directory
  ansible.builtin.file:
    path: /opt/todoapp
    state: directory
    owner: ec2-user
```